managers/muse.py

```diff
@@ -121,12 +121,24 @@ def constructWebpubManifest(self):
                 pdfManifest.closeSection()
 
             if pdfItem:
-                pdfManifest.addChapter(
-                    '{}{}'.format(
-                        self.MUSE_ROOT, pdfItem.parent.get('href')
-                    ),
-                    titleItem.span.a.string,
-                )
+                #skipCover indicates whether to skip the intersitital cover page by adding a query parameter
+                skipCover = False
+                if skipCover == False:
+                    pdfManifest.addChapter(
+                        '{}{}'.format(
+                            self.MUSE_ROOT, pdfItem.parent.get('href')
+                        ),
+                        titleItem.span.a.string,
+                    )
+                if skipCover == True:
+                    queryParam = '?start=2'
```

Comment:
I think this could be simplified a bit by just using `skipCover` to set the `queryparam` and have only one call to `addChapter`. But that's a preference thing!